systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/clientproperties/ConsumerProperties.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.kafkaclients.clientproperties;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.common.security.auth.SecurityProtocol;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ConsumerProperties extends AbstractKafkaClientProperties<ConsumerProperties> {
+
+    private static final Logger LOGGER = LogManager.getLogger(ConsumerProperties.class);
+
+    public static class ConsumerPropertiesBuilder extends AbstractKafkaClientProperties.KafkaClientPropertiesBuilder<ConsumerPropertiesBuilder> {
+
+        public ConsumerPropertiesBuilder withBootstrapServerConfig(String bootstrapServer) {
+
+            this.properties.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServer);
+            return this;
+        }
+
+        public ConsumerPropertiesBuilder withKeyDeserializerConfig(Class<? extends Deserializer> keyDeSerializer) {
+
+            this.properties.setProperty(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, keyDeSerializer.getName());
+            return this;
+        }
+
+        public ConsumerPropertiesBuilder withValueDeserializerConfig(Class<? extends Deserializer> valueDeSerializer) {
+
+            this.properties.setProperty(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, valueDeSerializer.getName());
+            return this;
+        }
+
+        public ConsumerPropertiesBuilder withGroupIdConfig(String groupIdConfig) {
+
+            this.properties.setProperty(ConsumerConfig.GROUP_ID_CONFIG, groupIdConfig);
+            return this;
+        }
+
+        public ConsumerPropertiesBuilder withClientIdConfig(String clientId) {
+
+            this.properties.setProperty(ConsumerConfig.CLIENT_ID_CONFIG, clientId);
+            return this;
+        }
+
+        @SuppressWarnings("Regexp") // for the `.toLowerCase()` because kafka needs this property as lower-case
```

Comment:
Why is this needed? You should use it like this ` .toLowerCase(Locale.ENGLISH)` I think you can remove `DM_CONVERT_CASE` as well after the change.